lib/util/win32-normalize-options.js

```diff
@@ -0,0 +1,108 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
```

Comment:
question: not sure what i should write here, every other file has author, should I add same author (`Author Tobias Koppers @sokra`) as other files?

Reply:
@davakh You can write own name here